public/pages/CreateRollup/utils/constants.ts

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+//Added last_updated_time and start_time to avoid error, but they should be system generated.
+//Added a delay as 0, but this should be optional, so null should be fine.
+export const EMPTY_ROLLUP = JSON.stringify({
+  rollup: {
+    continuous: false,
+    description: "",
+    dimensions: [
+      {
+        date_histogram: {
+          source_field: "",
+          fixed_interval: "2ms",
+          timezone: "Africa/Abidjan",
```

Comment:
A bit unusual choices for defaults. Let's change this to "1h" and UTC timezone

Reply:
Changed